web/styling.scss

```diff
@@ -5,6 +5,7 @@
   --foreground-color: #17191C;
   --light-color: #737D8C;
   --primary-color: #0DBD8B;
+  --primary-color-disabled: #0dbd8bb4;
```

Comment:
TIL: Browsers accept 4 byte hex colours to also set the alpha value.

Reply:
This seemed to be what vs-code suggested to me, but it might be super incompatible.